Frameworks/UIKit/UIRuntimeOutletCollectionConnection.mm

```diff
@@ -30,16 +30,18 @@ - (instancetype)initWithCoder:(NSCoder*)coder {
 
 - (void)_makeConnection {
     //  Grab existing collection from the target
-    NSMutableArray* collection = [source valueForKey:label];
+    if (source) {
```

Comment:
I definitely broke on this the other day but can't repro it anymore. Weird.

---
In reply to: [82630600](https://github.com/Microsoft/WinObjC/pull/1106#discussion_r82630600) [](ancestors = 82630600)